.github/workflows/Repology.yml

```diff
@@ -18,6 +18,7 @@ jobs:
       - name: Generate repology.json
         env:
           CREW_KERNEL_VERSION: 5.1
+          LIBC_VERSION: 2.37
```

Comment:
Why are these environment variables necessary?

Reply:
Otherwise the github actions runner reports a kernel and libc version we don't support, which causes some problems.